### PR TITLE
Updating Wazuh ruleset test scripts and UT with new standard

### PR DESCRIPTION
**Pull request by pereyra-m**
|Related issue|
|---|
|#791|

### Description

This PR updates the _file_test.py_ and _runtests.py_ python scripts, and all the related  examples with the current Wazuh standard.
It also adds new test cases for the new logs printed for the manager and agents.

### Tests

- [x] The _runtests.py_ successfully passes al the tests.

### CODE REVIEW

File: tools/rules-testing/tests/wazuh.ini [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+[wazuh: active response: delete firewall]
+log 4 pass = Sat May  7 03:27:57 CDT 2011 /INSTALLATION/PATH/active-response/bin/firewall-drop.sh delete - 172.16.0.1 1304756247.60385 31151
+
+rule = 602
+alert = 3
+decoder = ar_log
+
+[wazuh-logcollector: ignore informational messages at startup]
+log 5 pass = 2015/01/29 21:09:49 wazuh-logcollector(1950): INFO: Analyzing file: '/var/log/httpd/error_log'.
```
Comment (palaciosjeremias) [inline tools/rules-testing/tests/wazuh.ini:29]: Shouldn't we create the legacy version of this test as in `[wazuh: agent's logs: legacy started]`

File: tools/file-testing/file_test.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 # Requirements:
 #  Python 2.6 or later
-#  OSSEC 2.8 or later
+#  Wazuh 2.8 or later
```
Comment (palaciosjeremias) [inline tools/file-testing/file_test.py:12]: OSSEC V2.8 was released in 2014 and Wazuh started in 2015. The first documented version of Wazuh (2.1) was released in 2017.
I think we can eliminate this requirement.

File: tools/rules-testing/tests/wazuh.ini (hunk 1 above)
Comment (pereyra-m) [reply tools/rules-testing/tests/wazuh.ini:29]: A new test was created!

File: tools/file-testing/file_test.py (hunk 2 above)
Comment (pereyra-m) [reply tools/file-testing/file_test.py:12]: You are right. Removed!

**Review by palaciosjeremias — APPROVED** [review]
LGTM

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**